src/main/java/core/basesyntax/action/BalanceHandler.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.action;
+
+public class BalanceHandler implements ActionHandler {
+    @Override
+    public int performAction(int amount) {
+        return amount;
+    }
```

Comment:
I understood completely nothing. 

Why do we need storage at all? 
What is FruitTransaction? What it does? 
Why do you say, that ActionHandlers are invalid, if they perform their main function: get positive or negative value, depending on the Action value?

Reply:
we need storage because it's required by the task. look closely at Fruit DB
![image](https://github.com/mate-academy/jv-fruit-shop/assets/73041091/f5707801-914a-4237-b896-171c154ba8bc)

your ActionHandlers are invalid because they not affect values
FruitTransaction is just recomendation, but it would work:
![image](https://github.com/mate-academy/jv-fruit-shop/assets/73041091/dfdaba06-9f37-4ac5-9b26-91b3435c9ea2)

at first, add Storage to the project and use it. if you'll have more questions, feel free to ask in the chat or come to the QnA. better not to communicate here, it's unconvenient